client/js/entry-client-hydrogen.js

```diff
@@ -0,0 +1,9 @@
+import 'matrix-public-archive-shared/hydrogen-vm-render-script';
```

Comment:
The client scripts don't load in the browser. Trying to fix this up in https://github.com/matrix-org/matrix-public-archive/pull/175